### PR TITLE
Feat/tvl

### PR DESCRIPTION
**Pull request by leonprou**
## Proposed changes

- building a job to calculate daily TVL

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: server/src/utils/token/index.js [hunk 1]
```diff
@@ -30,11 +31,19 @@ const fetchTokenData = async (address, fields = {}, web3 = foreign.web3) => {
 
 const fetchBalance = async ({ createContract }, tokenAddress, accountAddress) => {
   const tokenContract = createContract(BasicTokenAbi, tokenAddress)
+  console.log({ tokenAddress, accountAddress })
```
Comment (LiorAgnin) [inline server/src/utils/token/index.js:34]: Remove the logs

File: server/src/utils/token/index.js [hunk 2]
```diff
@@ -30,11 +31,19 @@ const fetchTokenData = async (address, fields = {}, web3 = foreign.web3) => {
...
   const accountBalance = await tokenContract.methods.balanceOf(accountAddress).call()
-  console.log(`balance of account ${accountAddress} of token ${tokenAddress} is ${accountBalance}`)
+  // console.log(`balance of account ${accountAddress} of token ${tokenAddress} is ${accountBalance}`)
```
Comment (LiorAgnin) [inline server/src/utils/token/index.js:36]: also here

**Review by LiorAgnin — APPROVED** [review]
Looks good!.
If you can remove the logs that would be nice

**Comment by leonprou** [comment]
@LiorAgnin Thanks, done 👍

**Review by LiorAgnin — APPROVED** [review]

**Pull request merged**